app/src/main/java/io/gnosis/safe/ui/transactions/details/view/LabeledNamedAddressItem.kt

```diff
@@ -0,0 +1,40 @@
+package io.gnosis.safe.ui.transactions.details.view
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import io.gnosis.safe.databinding.ViewLabeledNamedAddressItemBinding
+import pm.gnosis.model.Solidity
+
+
+class LabeledNamedAddressItem @JvmOverloads constructor(
```

Comment:
Shouldn't we be able to use this in a lot more places?

Reply:
please elaborate on this